src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/LoggerExtensions.cs

```diff
@@ -15,11 +15,17 @@ namespace Microsoft.Extensions.Logging
     /// </summary>
     public static class LoggerExtensions
     {
+        private static readonly Action<ILogger, int, Exception> _logFunctionRetryFailed =
+           LoggerMessage.Define<int>(
+           LogLevel.Error,
+           new EventId(324, nameof(LogFunctionRetryFailed)),
+           "Function failed after '{attempt}' retries.");
```

Comment:
nit: "Function execution failed..." to match the wording used in the other log below.